aioredis/pool.py

```diff
@@ -29,18 +30,20 @@ def create_pool(address, *, db=None, password=None, ssl=None, encoding=None,
 
     All arguments are the same as for create_connection.
 
-    Returns RedisPool instance.
+    Returns RedisPool instance or a pool_cls if it is given.
     """
     if commands_factory is not _NOTSET:
         warnings.warn(
             "commands_factory argument is deprecated and will be removed!",
             DeprecationWarning)
 
-    pool = ConnectionsPool(address, db, password, encoding,
-                           minsize=minsize, maxsize=maxsize,
-                           ssl=ssl, parser=parser,
-                           create_connection_timeout=create_connection_timeout,
-                           loop=loop)
+    cls = pool_cls or ConnectionsPool
```

Comment:
done, thanks.

Btw can you restart the Travis Job for an unknown reason the py35 environment failed the command `make examples`?